libopenage/pathfinding/pathfinder.cpp

```diff
@@ -246,7 +250,7 @@ const Pathfinder::portal_star_t Pathfinder::portal_a_star(const PathRequest &req
 	while (not node_candidates.empty()) {
 		auto current_node = node_candidates.pop();
 
-		current_node->was_best = true;
+		was_best.insert(current_node->portal->get_id());
```

Comment:
Why did you make `was_best` a set instead of storing it in the node? Just curious.

Reply:
Because the nodes would have to be reset after each call to portal_a_star, which would be inefficient. As far as I can tell anyway.

Reply:
Good point.